tools/ci_build/coverage.py

```diff
@@ -39,11 +39,12 @@ def main():
     script_dir = os.path.realpath(os.path.dirname(__file__))
     source_dir = os.path.normpath(os.path.join(script_dir, "..", ".."))
     cwd = os.path.abspath(os.path.join(args.build_dir, args.config))
-    adb_shell('cd /data/local/tmp && tar -zcvf gcda_files.tar.gz *.dir')
+    adb_shell('cd /data/local/tmp && tar -zcf gcda_files.tar.gz *.dir')
     adb_pull('/data/local/tmp/gcda_files.tar.gz', cwd)
     os.chdir(cwd)
-    run_subprocess("tar -zxvf gcda_files.tar.gz -C CMakeFiles".split(' '))
-    run_subprocess("gcovr -s -r {} .".format(os.path.join(source_dir, "onnxruntime")).split(' '),
+    run_subprocess("tar -zxf gcda_files.tar.gz -C CMakeFiles".split(' '))
+    run_subprocess("gcovr -s -r {} . -o {}".format(os.path.join(source_dir, "onnxruntime"),
+                   os.path.join(cwd, "coverage_rpt.txt")).split(' '),
```

Comment:
what if cwd or source_dir contain spaces? consider directly passing a list of strings to run_subprocess